test_data/duplicates/shapes.txt

```diff
@@ -0,0 +1,30 @@
+shape_id,shape_pt_lat,shape_pt_lon,shape_pt_sequence,shape_dist_traveled
```

Comment:
Do we have a test file (or maybe we can include it here) where we have no duplicates (with `(shape_id, shape_pt_sequence)`) but multiple lines for the same `shape_id`

Reply:
There is in the [test file](https://github.com/etalab/transport-validator/blob/4c23a1814ad565df90a4b2dcc698721bbf245951/test_data/duplicates/shapes.txt), B_shp has 2 points for different pt_sequences, and A_shp has 4 points, with only 2 being duplicated

Reply:
no worry, it's nice to check thoroughly this, I did not realize there were potential alerting email on mistake on the validator :sweat_smile: